### DISCUSSION TITLE
How to workaround web tool's 'row limit' ?

### DISCUSSION THREAD

**Discussion by jonathanjuhn** [apache/superset#13993] [Q&A / Help]
Hey all, 

I'm a non-technical superset user with a question around exporting CSVs with ~1-2M rows. 

The max row limit for the Web version is 100,000. I suspect this is b/c of some browser resource limitation. 

Is there a way for me to override this limit - perhaps by including some functions to directly download as a CSV? 

![image](https://user-images.githubusercontent.com/56653788/113898313-bfe24680-9799-11eb-8400-ca1f56fb0703.png)

edited: screenshots

**Comment by maltoze** [comment]
https://github.com/apache/superset/blob/b5c95c537c6cefb3e36eac8b82625565a5152a8a/superset/config.py#L605
change this value to `200000` should be work

**Reply by jonathanjuhn** [reply]
Thanks for the pointer. 
Flagged this solution to my Eng team and they mentioned this would require a much larger re-deploy of superset which is a little too heavy for what I'm ultimately trying to achieve.
